docs/app_modules/interchain-accounts/ica_auth.md

```diff
@@ -210,7 +210,13 @@ packetData := icatypes.InterchainAccountPacketData{
     Data: data,
 }
 
-_, err = keeper.icaControllerKeeper.TrySendTx(ctx, chanCap, p, packetData)
+// Obtain timeout timestamp
+// An appropriate timeout timestamp must be determined based on the usage of the interchain account.
+// If the packet is timed out, the channel will be closed requiring a new channel to be created 
+// and reassociated with the existing interchain account on the host chain. 
```

Comment:
```suggestion
```

Reply:
I think we should remove this line, I feel like it overcomplicates it a bit. I guess we're handling this under the hood.